sorbet/rbi/shims/uri.rbi

```diff
@@ -16,4 +16,6 @@ module URI
     sig { returns(String) }
     attr_reader :fragment
   end
+
+  class WS; end
```

Comment:
I opened a PR to add this to Sorbet's payload: https://github.com/sorbet/sorbet/pull/6673 :+1: